api/envoy/data/accesslog/v3/accesslog.proto

```diff
@@ -183,6 +183,9 @@ message AccessLogCommon {
   // state serialized to any message other than `google.protobuf.Any` it will be packed into
   // `google.protobuf.Any`.
   map<string, google.protobuf.Any> filter_state_objects = 21;
+
+  // A list of custom tags with unique tag name to create tags for the logs.
```

Comment:
Could you elaborate on how "tag" means here? new comer won't understand what this actually does. Maybe better add explanation annotation, etc?